.travis.yml

```diff
@@ -9,6 +9,7 @@ go:
   # TODO jiayu: re-evaluate the pending issues and add back `gotype`
   - 1.8
   - 1.9
+  - "1.10"
```

Comment:
Why the quotes here abut not above?

Reply:
https://github.com/travis-ci/travis-ci/issues/9247